tests/environments/minimal.lua

```diff
@@ -11,7 +11,10 @@ function M.setup(root_env) end
 function M.init()
   vim.cmd([[set runtimepath=$VIMRUNTIME]])
   vim.opt.runtimepath:append(M.root())
-  vim.opt.packpath = { M.root(".tests/minimal/site") }
+  vim.opt.packpath = {
+    M.root(".tests/minimal/site"),
+    M.root("tests"),
```

Comment:
This makes it possible to require files in the tests folder.

Reply:
o.O wow